addons/point_of_sale/static/src/overrides/action_service.js

```diff
@@ -0,0 +1,22 @@
+import { patch } from "@web/core/utils/patch";
+import { actionService } from "@web/webclient/actions/action_service";
+
+patch(actionService, {
+    start(env) {
+        // The action service wants to open the majority of links in the main container, not in new dialog.
+        // The problem is that in pos we don't have the main container, so the links are simply not opened.
+        // This is a workaround to always open links in new dialogs.
+        return {
+            ...super.start(env),
+            doAction: async (actionRequest, options = {}) => {
+                if (
+                    document.body.classList.contains("modal-open") &&
+                    typeof actionRequest === "object"
+                ) {
+                    actionRequest.target = "new";
+                }
+                return super.start(env).doAction(actionRequest, options);
```

Comment:
Calling `super.start` at every `doAction` will call the side-effects of `makeActionManager`.

You should probably do something like this:

```js
const baseActionService = super.start(env);
return {
    ...baseActionService,
    doAction: async () => {
        // ... <your custom logic> ...
        return baseActionService.doAction(...arguments);
    }
}
```